src/pages/sepa.js

```diff
@@ -81,7 +82,16 @@ var SEPA = React.createClass({
           <div className="submit-button">
             <button className="donate-submit submit-btn" onClick={e => this.handleSubmit(e)}>Submit</button>
           </div>
-      </form>
+        </form>
+        <div className="sepa-mandate">
```

Comment:
can we add some styling (bold or something) to the text so that it's a little more prominent? Based on what I can tell, they are pretty strict about the mandate (we actually need to send it to the donor via email later too) so it would be good to make it a little more obvious.

Reply:
Can you link me to the regulations? Making it bold will make the form look like shit.

Reply:
https://stripe.com/docs/sources/sepa-debit#prerequisite is where they mention it.

Reply:
Yeah, I think it will be OK! But let's see what legal says and we can adjust pronto.